pkg/skaffold/deploy/helm.go

```diff
@@ -138,7 +142,9 @@ func (h *HelmDeployer) Deploy(ctx context.Context, out io.Writer, builds []build
 
 	event.DeployComplete()
 
-	labelDeployResults(h.labels, dRes)
+	if h.addLabels {
```

Comment:
When AddSkaffoldLabels is false, we are supposed to still add the custom labels. Is that the case here?

Reply:
I didn't know this actually. I ignored the "skaffold" part and just took this as "should we add labels or not". updated, and also added some unit tests for adding custom labels in render and deploy since those would have caught this.